controller.go

```diff
@@ -1042,10 +1043,15 @@ func (c *controller) NewSandbox(containerID string, options ...SandboxOption) (S
 	}
 	c.Unlock()
 
+	sandboxID := stringid.GenerateRandomID()
+	if runtime.GOOS == "windows" {
+		sandboxID = containerID
+	}
+
 	// Create sandbox and process options first. Key generation depends on an option
 	if sb == nil {
 		sb = &sandbox{
-			id:                 stringid.GenerateRandomID(),
+			id:                 sandboxID,
```

Comment:
Quick clarification: I assume the `sb.id = "ingress_sbox"` below does not affect Windows?

Reply:
No it shouldn't affect windows

Reply:
ping @ddebroy

Reply:
thanks for clarifying .. LGTM.